packages/examples/src/canvas-operations-alpha-matte.meta

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 0,
+  "shared": {
+    "background": null,
+    "range": [
+      0,
+      null
+    ],
+    "size": {
+      "x": 1920,
+      "y": 1080
+    },
+    "audioOffset": 0
+  },
+  "preview": {
+    "fps": 30,
+    "resolutionScale": 1
+  },
+  "rendering": {
+    "fps": 30,
```

Comment:
Shouldn't the default value be `60`?
Is there any reason for it to be `30` in these examples?

Reply:
Same reason I set the default rendering resolution from 2x to back to 1.

Trying to run 2 or more animations made them extremely unresponsive and stuttery.

Reply:
Even after removing the codeblock tweening?

Reply:
Yep. This was after changing to a static code block.

Animations would play back extremely slow until I reduced the render size (which was at 2x by default for some reason) and framerate.

You can still notice that it slows down a little bit when playing multiple animations, but it's no longer that big of a change.

Reply:
For me, on Chrome, in 4k 60fps, none of the examples cause trouble. Maybe it's a hardware difference though.